app/views/podcasts/index.html.erb

```diff
@@ -1,26 +1,51 @@
 <% content_for :title, "My Podcasts" %>
 
-<p style="color: green"><%= notice %></p>
-<div class="row mt-4 mb-4">
-  <div class="dashboard header">
-    <div class="col d-flex align-items-center ">
-      <h1 class="text-black fw-bold me-4 mb-0"><%= t(".my_podcasts") %></h1>
-
-      <a data-bs-toggle="dropdown">
-      <button class="btn btn-outline-light btn-sm dropdown-toggle">
-        Sort <span class="material-icons" style="color: green" aria-hidden="true" aria-hidden="true">sort</span>
-      </button>
-      <ul class="dropdown-menu">
-        <li><%= link_to "Recent Activity", podcasts_path(sort: "Recent"), class: "dropdown-item active" %></li>
-        <li><%= link_to "# of Episodes", podcasts_path(sort: "episode_count"), class: "dropdown-item" %></li>
-        <li><%= link_to "A-Z", podcasts_path(sort: "A-Z"), class: "dropdown-item" %></li>
-        <li><%= link_to "Z-A", podcasts_path(sort: "Z-A"), class: "dropdown-item" %></li>
-      </ul>
+<div class="container p-0">
+  <p style="color: green"><%= notice %></p>
+  <div class="d-flex align-items-center justify-content-between">
+    <div class="dashboard header">
+      <div class="d-grid d-lg-flex gap-2 align-items-center">
+        <h1 class="text-black fw-bold d-flex flex-fill my-4"><%= t(".my_podcasts") %></h1>
+
+        <a data-bs-toggle="dropdown">
+        <button class="btn btn-outline-light btn-sm dropdown-toggle">
+          Sort <span class="material-icons" style="color: green" aria-hidden="true" aria-hidden="true">sort</span>
+        </button>
+        <ul class="dropdown-menu">
+          <li><%= link_to "Recent Activity", podcasts_path(sort: "Recent"), class: "dropdown-item active" %></li>
+          <li><%= link_to "# of Episodes", podcasts_path(sort: "episode_count"), class: "dropdown-item" %></li>
+          <li><%= link_to "A-Z", podcasts_path(sort: "A-Z"), class: "dropdown-item" %></li>
+          <li><%= link_to "Z-A", podcasts_path(sort: "Z-A"), class: "dropdown-item" %></li>
+        </ul>
+      </div>
+    </div>
+
+    <div class="row-reverse col-4">
+        <%= form_with(
+              url: podcasts_path,
+              method: :get,
+              data: {
+                controller: "search",
+                "turbo-frame": "podcasts",
```

Comment:
Oh, and I think you can simplify this?  If you change this to `turbo_frame: "podcasts"` ... Rails will do-the-right-thing, and put `data-turbo-frame="podcasts"` (with the underscore changed to a dash) in the HTML.

And the `turbo-action` isn't necessary, as "replace" is already the default.  `method: :get` is also a default.  So that trims this down to a more-readable-single-line:

```html
<%= form_with(url: podcasts_path, data: {controller: "search", turbo_frame: "podcasts"}) %>
```